content/manager_webui/overview.md

```diff
@@ -61,3 +61,11 @@ The UI templates are the sets of pages presented to the users upon logging into
 - Local Blueprint Catalog
 - Deployments
 - System Resources
+
+## Advanced Configuration
+
+In `/opt/cloudify-stage/dist/userData/userConfig.json` on Cloudify Manager machine you can perform more advanced customization of Cloudify Console.
```

Comment:
Avoid "perform" unless you want to sing
perform > configure

Reply:
Done. BTW - there are two similar usages of `perform` in this doc. One is yours ;-)